Assignment6/question2.js

```diff
@@ -0,0 +1,43 @@
+/*
+    2. What makes a method mutating or non mutating in Javascript? Find out whether each of the following methods are mutating or non-mutating. How can you verify this?:
```

Comment:
where is the ans to last que?

" How can you verify this? "

Reply:
done